states/WI/governor.yaml

```diff
@@ -56,6 +56,8 @@ contact_form:
     - wait:
         - value: 5
     # Confirmation screen.
+    - javascript:
+        - value: "document.querySelector('#hdn-f1d59b73-aee4-469a-9e8a-38efdaca963e').click()"
```

Comment:
Does the `click_on` need to be removed?

Reply:
No.  The flow is like this:
Fill out form -> click "next" -> check confirm box -> click next -> message sent.

Reply:
👍